### DISCUSSION TITLE
Keyboard Navigation

### DISCUSSION THREAD

**Discussion by jcorporation** [jcorporation/myMPD#497] [General]
Does somebody use the shortcuts or keyboard navigation?

**Comment by DoDiesis** [comment]
As I mainly use myMPD with my mobile phone, to control my own music streamer, I never use the keyboard navigation.

**Comment by xero-timothy** [comment]
Personally I am finding it very frustrating that command + C overrides _copy_ and clears my play queue! Is it possible to configure or disable these shortcuts?

**Reply by jcorporation** [reply]
That’s a bug and should b fixed with 39f3fee1.

**Reply by xero-timothy** [reply]
Amazing! Thank you so much :)
